pages/wiki/technical.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Technical Details
+layout: documentation
+---
+This page contains technical details about currently supported watches.
+
+# AsteroidOS Watch Technical Details
+---
+
+While AsteroidOS does not use Android, it does rely on drivers that are be supported with [libhybris](https://github.com/libhybris/libhybris).
+
+### Supported watches
+
+<table class="table table-striped">
+  <tr>
+  <thead>
+    <th>Model Name</th>
+    <th>Codename</th>
+    <th>Support</th>
+    <th>SOC</th>
+    <th>Kernel</th>
+    <th>original Android</th>
```

Comment:
Original Android, with uppercase start imo.
And i am not totally sure if it fits perfectly, but "Stock Android" might be shorter .
Afaiu, the term Stock Android describes the android version supplied by the manufacturer or google and is often used to refer to the baseline or reference version of the operating system.

Reply:
I put "Original Android" and also added a link to the part of the FAQ that describes the relationship to Android drivers in more detail.